modules/reklamstoreBidAdapter.js

```diff
@@ -0,0 +1,141 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'reklamstore';
+const ENDPOINT_URL = '//ads.rekmob.com/m/prebid';
+const CURRENCY = 'USD';
+const TIME_TO_LIVE = 360;
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['reklamstore'],
```

Comment:
No need to add aliases if you are repeating bidder_code here. aliases are to be used if you want to use some short code for bidder in adUnit config.